apps/core/scenes/results/Results.js

```diff
@@ -52,9 +52,16 @@ type Props = {|
   +bags: number,
   +sortBy: string,
   +layout: number,
+  +routerQuery: Object,
```

Comment:
Can you create a type for it. We should know what params we can get

Reply:
Yes, I put it as `SearchParameter`